README.md

```diff
@@ -34,6 +34,7 @@ Note for internal use only: See the [technical specifications](https://github.co
 
 ### Implemented smart contracts
 
+- `cancelComputePlan`
```

Comment:
is this related to this PR?

Reply:
Yes and no: its because I updated this list with the missing smart contract.